packages/coinstac-ui/app/main/utils/boot/parse-cli-input.js

```diff
@@ -27,7 +27,7 @@ function parse() {
         data = data.toString();
         // don't fill the screen with useful build information. scan for it,
         // summarize it instead
-        const builtRegex = /.*(\[(built|not cacheable)\]|\{0\}).*[\r\n]/gm;
+        const builtRegex = /.*(\[(built|not cacheable)]|\{0\}).*[\r\n]/gm;
```

Comment:
I guess that `\` isn’t necessary when the first `[` is escaped? Linter barked at it.